diff -ruN popt-1.10.2/popthelp.c popt-1.10.2-linebreak/popthelp.c
--- popt-1.10.2/popthelp.c	2004-02-26 02:20:53.000000000 +0100
+++ popt-1.10.2-linebreak/popthelp.c	2005-07-24 15:37:11.000000000 +0200
@@ -375,6 +375,18 @@
     while (helpLength > lineLength) {
 	const char * ch;
 	char format[16];
+        
+        /* this code will find newline in the description and align it */
+        ch = help;
+	while (ch < help + lineLength && *ch != '\n') ch++;
+	if (*ch == '\n')
+        {
+            sprintf(format, "%%.%ds\n%%%ds", (int) (ch - help), (int) indentLength);
+            fprintf(fp, format, help, " ");
+            help = ch+1;
+            helpLength = strlen(help);
+            continue;
+        }
 
 	ch = help + lineLength - 1;
 	while (ch > help && !isspace(*ch)) ch--;
@@ -392,6 +404,24 @@
     }
 /*@=boundsread@*/
 
+    const char * ch;
+    char format[16];
+    ch = help;
+    while (ch < help + helpLength)
+    {
+        if (*ch == '\n')
+        {
+           sprintf(format, "%%.%ds\n%%%ds", (int) (ch - help), (int) indentLength);
+           fprintf(fp, format, help, " ");
+           ch++;
+           if (ch == help + helpLength)
+               fprintf(fp, "\n");
+           help = ch;
+           helpLength = strlen(help);
+        }
+        else
+           ch++;
+    }
     if (helpLength) fprintf(fp, "%s\n", help);
 
 out:
